src/z_generate_repo.prog.abap

```diff
@@ -458,7 +458,10 @@ CLASS lcl_generator IMPLEMENTATION.
         dyname     = sy-repid
         dynumb     = sy-dynnr
       TABLES
-        dynpfields = dynpfields.
+        dynpfields = dynpfields
+      EXCEPTIONS
+        OTHERS     = 1.
```

Comment:
You could also set the return value to `0` if you just want to ignore any exception (fire and forget)

```suggestion
        OTHERS     = 0.
```